RELEASE-NOTES.txt

```diff
@@ -1,6 +1,6 @@
 17.3
 -----
-
+* [**] Fix issue where Undo would sometimes convert content to the the classic editor. [#16342]
```

Comment:
```suggestion
* [**] Fix issue where Undo would sometimes convert content to the classic editor. [#16342]
```

Reply:
I wonder if this needs to be more specific since Undo could refer to the block editor undo button. Also, might be worth clarifying that it would "sometimes make posts/pages lose their block formatting", if that makes sense – up to you. Not sure if there's a way of being more specific without getting too technical.
When I viewed the HTML of a post where this happened, it looks like blocks lose their block delimiters, but that's too technical for the release notes.

Reply:
>Not sure if there's a way of being more specific without getting too technical.

This was my fear. If I elaborated on the steps it would probably be a bit too technical. I agree that Undo isn't clear enough though. I'll re-word this line.

> When I viewed the HTML of a post where this happened, it looks like blocks lose their block delimiters, but that's too technical for the release notes.

Yeah, the app uses the presence of one formatted block to decide if we should use the classic editor or not.

Reply:
Ok, sounds good!